addons/switch_focus/box_list.lua

```diff
@@ -0,0 +1,51 @@
+local box_list = {
+    _orderd = {},
+    _byValue = {},
```

Comment:
You use `snake_case` for everything else, but `camelCase` here. Would prefer to `snake_case` this as well.